lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1349,7 +1349,7 @@ SwiftExpressionParser::ParseAndImport(
   if (enable_bare_slash_regex_literals) {
     lang_opts.enableFeature(swift::Feature::BareSlashRegexLiterals);
   }
-  if (uint32_t version = m_expr.Language().version) {
+  if (uint32_t version = m_expr.Language().version)
```

Comment:
This change looks scary.

Reply:
But I think it's correct.

Reply:
Heh, yeah. I was going to add the `}` instead but this matches what was... somewhere (stable/20230725 I assume?)